src/organisms/cards/PolicyCard/policy_card.module.scss

```diff
@@ -0,0 +1,127 @@
+$border: #dfdfdf;
+$footer: #f6f6f6;
+
+@mixin section($grow, $centered) {
+  flex: #{$grow} 1 0;
+  padding: 0 ru(1.25);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+
+  @if $centered {
+    align-items: center;
+  }
+}
+
+.policy-card {
+  border: 1px solid $border;
+  border-radius: ru(.25);
+  max-width: rem-calc(1060px);
```

Comment:
@cisacke obviously this was intentional, I just don't see anything about `max-width` for the card in the zeplin, wanted to double check

Reply:
👍 confirmed with Jim, we're going to have max-width of 1200px